database/queries/unlike_career.js

```diff
@@ -0,0 +1,13 @@
+const dbConnection = require('../db_connection');
+
+const unlikeCareer = (data, callback) => {
```

Comment:
I think the function could be a bit clearer and consistent with the others if instead of taking a data array it took `userId, careerId, callback` 

I guess the `add_career` query has a data array so you could have it either way. In this case it's probably clearer with multiple arguments but I'll leave it to you to decide.